build.gradle

```diff
@@ -136,7 +136,7 @@ configurations.all {
         force "joda-time:joda-time:${versions.joda}"
         force "com.fasterxml.jackson.core:jackson-core:2.13.4"
         force "commons-logging:commons-logging:${versions.commonslogging}"
-        force "org.apache.httpcomponents:httpcore:${versions.httpcore}"
+        force "org.apache.httpcomponents:httpcore5:${versions.httpcore5}"
```

Comment:
Could we possible just remove this `force`? and just inherit it as is from opensearch core?

Reply:
do you know what's the initial reason of adding this `force` instead of inheriting it from opensearch core?

Reply:
Not sure, seems to be with our initial set up of AD. Usually from my understanding its used if we need a specific version of a dependency that doesn't align with core so we force it to be something else. I believe though since we aren't in need of anything specific we should be able to remove this line and build/testing should be fine.

Reply:
Before doing that I would confirm with @ylwu-amzn or @kaituo on the reasoning. When that's figured out, if it's still needed, we should add a clarifying comment above.